app/Lib/Rscripts/analyse.R

```diff
@@ -94,9 +136,14 @@ Analyse <- function(key, input.answers, number.answeroptions) {
       percentage.answer.options <- 0
       corrected.item.tot.cor.answ.option <- 0
     }
-    
-    # Computes Cronbach's Alpha
-    cronbach <- round(cronbach(input.correct)$alpha, digits = 3)
+
+    # Computes Cronbach's Alpha for overall test
+    # If there is no variance in the total score returns -100
+    if(!all(rowSums(input.correct) == rowSums(input.correct)[1])){
```

Comment:
I did, but we do not follow it on two points, resulting in so many warnings that I do not bother anymore. The first is that the linter package follows a different guide which forbids '.' in variable names, while the google guide encourages it. The second is that our variable names are too long. I think we agreed a while back on not changing the names for consistency. I kinda lost sight of the forest (other style errors) behind the trees (these two erros). However, I did use it again and I hope it is okay now.